internal/remote/user.go

```diff
@@ -52,7 +48,7 @@ func newUser(userID, path string, conn connector.Connector) (*User, error) {
 		path:              path,
 		conn:              conn,
 		updatesCh:         make(chan imap.Update),
-		queue:             pchan.New[operation](),
+		mergeQueue:        newUserOpQueue(queue.NewTQueue[operation]()),
```

Comment:
why is it called mergeQueue? why not just simply `opQueue`?

Reply:
Done.